newIDE/app/src/MainFrame/index.js

```diff
@@ -1789,6 +1791,7 @@ class MainFrame extends React.Component<Props, State> {
       useStorageProvider,
       i18n,
       renderGDJSDevelopmentWatcher,
+      renderMainMenu: MainMenu,
```

Comment:
As a rule of thumb, if you have to rename a prop, it's a red flag that something is wrong ;) 

Here is some interesting read: https://reactjs.org/docs/render-props.html